tests/regression/run-api.sh

```diff
@@ -21,19 +21,24 @@ for ((i = 0; i < ${#OPTIONS[@]}; i++)); do
 	option=${OPTIONS[${i}]}
 	cli_arg=$(echo ${option} | cut -d '|' -f 2)
 	build_arg=$(echo ${option} | cut -d '|' -f 1)
+	socket=$(mktemp)
 
-	bazel run accelerator ${build_arg} -- --ta_port=${TA_PORT} ${cli_arg} --proxy_passthrough &
+	bazel run accelerator ${build_arg} -- --ta_port=${TA_PORT} ${cli_arg} --proxy_passthrough --socket=${socket} &
 	TA=$!
 	trap "kill -9 ${TA};" INT # Trap SIGINT from Ctrl-C to stop TA
 
-	# Wait until tangle-accelerator has been initialized
-	echo "==============Wait for TA starting=============="
-	while read -r line; do
-		if [[ "$line" == "$start_notification" ]]; then
-			echo "$line"
-		fi
-	done <<<$(nc -U -l $socket | tr '\0' '\n')
-	echo "==============TA has successfully started=============="
+	# if tangle-accelerator takes more than 30 secs to initialize, then exit and return failure.
+	timeout 30 bash tests/regression/ta_waiting.sh ${socket}
```

Comment:
Use `tests/regression/ta_waiting.sh` rather than `bash tests/regression/ta_waiting.sh` because `bash` might be found in unexpected PATH.